src/webview/MessageList.js

```diff
@@ -309,8 +285,16 @@ class MessageList extends Component<Props> {
     // https://github.com/react-native-community/react-native-webview/pull/697
     return (
       <WebView
-        source={{ baseUrl, html }}
-        originWhitelist={['file://']}
+        source={{ baseUrl: baseUrl.toString(), html }}
```

Comment:
Hmm, I get a Flow error on this line!

```
Error ┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈┈ src/webview/MessageList.js:288:17

Could not decide which case to select. Since case 1 [1] may work but if it doesn't
case 2 [1] looks promising too. To fix add a type annotation to call of method
toString [2].

     src/webview/MessageList.js
     285│     // https://github.com/react-native-community/react-native-webview/pull/697
     286│     return (
     287│       <WebView
 [2] 288│         source={{ baseUrl: baseUrl.toString(), html }}
     289│         originWhitelist={[
     290│           // `baseUrl.origin` may be "null" (an "opaque origin") for
     291│           // `file://` URLs. [1] [2] `react-native-webview` doesn't

     flow-typed/react-native-webview_v8.x.x.js
 [1] 495│     source?: WebViewSource,



Found 1 error
```

Puzzling that I see that even though it seems to have passed in CI. If you're not seeing the error, let's debug in chat.

Reply:
> it seems to have passed in CI

Ah, I think CI isn't a thing anymore, hence https://github.com/zulip/zulip-mobile/issues/4174?

Reply:
Ah indeed :slightly_smiling_face: (and fixed now.)

Reply:
I see the fix was:
```
        source={{ baseUrl: (baseUrl.toString(): string), html }}
```
i.e. adding an explicit cast to `string`. Weird that that makes a difference -- this method's return type is shown as simply `string` already. Seems like a Flow bug.

But the behavior with that workaround seems like the right behavior, so :shrug: